src/App.tsx

```diff
@@ -1,61 +1,132 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './components/types/Todo';
 
 export const App = () => {
+  const newTodos: Todo[] = todosFromServer.map(todo => ({
+    ...todo,
+    user:
+      usersFromServer.find(
+        userFromServer => todo.userId === userFromServer.id,
+      ) || null,
+  }));
+  const [todos, setTodos] = useState(newTodos);
+  const [title, setTitle] = useState('');
+  const [titleTouched, setTitleTouched] = useState(true);
```

Comment:
The initial state for `titleTouched` should be `false` instead of `true`, as the title input has not been interacted with initially. This will ensure that the error message is not shown prematurely.